src/components/Books.js

```diff
@@ -16,11 +16,29 @@ const Books = (props) => {
 
   return (
     <li>
-      <div>
-        <p>{bookType}</p>
-        <p>{name}</p>
+      {/* <div className="allbook-style"> */}
+      <div className="book-detail">
+        <div className="book-type-name">
+          <p className="book-type">{bookType}</p>
+          <p className="book-name">{name}</p>
+        </div>
+        <button type="button" className="remove-button" onClick={() => removeBook(book)}>remove</button>
       </div>
```

Comment:
- [ ] Features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented please add  what is missing
![ms](https://user-images.githubusercontent.com/44745855/145631348-acd1c6ed-cf33-403f-a00c-f193a104592c.png)